### PR TITLE
tar: add Chinese translation

### PR DESCRIPTION
**Pull request by hmsjy2017**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

 - **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/tar.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# tar
+
+> 归档实用程序。
+> 通常与压缩方法结合使用，例如 gzip 或 bzip2。
```
Comment (blueskyson) [inline pages.zh/common/tar.md:4]: I know this is somewhat strange in Chinese, but it's the current practice in tldr to use half-width punctuation when the tail of a sentence is in English.
```suggestion
> 通常与压缩方法结合使用，例如 gzip 或 bzip2.
```

File: pages.zh/common/tar.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+> 更多信息：<https://www.gnu.org/software/tar>
```
Comment (blueskyson) [inline pages.zh/common/tar.md:5]:
```suggestion
> 更多信息：<https://www.gnu.org/software/tar>.
```

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution. Just 2 little things to fix.

**Comment by hmsjy2017** [comment]
Done.

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**